resources/compass/charts/connectivity-adapter/templates/deployment.yaml

```
@@ -0,0 +1,31 @@
+apiVersion: apps/v1beta2
```

```suggestion
apiVersion: apps/v1
```
`apps/v1beta2` is deprecated and is not supported from Kubernetes 1.16 up.

---

thanks @Szymongib, I will solve this in another PR